app/connector/jira/src/main/java/io/syndesis/connector/jira/JiraComponentProxyFactory.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.connector.jira;
+
+import java.util.Map;
+import java.util.Optional;
+
+import io.syndesis.integration.component.proxy.ComponentDefinition;
+import io.syndesis.integration.component.proxy.ComponentProxyComponent;
+import io.syndesis.integration.component.proxy.ComponentProxyFactory;
+import org.apache.camel.Component;
+
+public class JiraComponentProxyFactory implements ComponentProxyFactory {
+
+    @Override
+    public ComponentProxyComponent newInstance(String componentId, String componentScheme) {
+        return new ComponentProxyComponent(componentId, componentScheme) {
+            @Override
+            protected Optional<Component> createDelegateComponent(ComponentDefinition definition, Map<String, Object> options) {
+                for (String k: options.keySet()) {
+                    String value = options.get(k).toString();
+                    // envelope the JQL in RAW() if it isn't
+                    boolean notRaw = value != null && value.length() > 4 && !"RAW(".equals(value.substring(0, 4));
+                    if ("jql".equals(k) && notRaw) {
+                        value = "RAW(" + value + ")";
```

Comment:
You should be able to set the option `raw` from the connector json

Reply:
Fixed that, thanks for the tip.